build/workspace.bzl

```diff
@@ -0,0 +1,73 @@
+# only native extensions are allowed in this file. For repository
+# initialization use :repositories.bzl. This is to avoid circular
```

Comment:
that part is a little confusing, though I'm not sure there's much of a way around it.